contracts/test/facades/PairBasedStrategyLibFacade.sol

```diff
@@ -141,7 +141,7 @@ contract PairBasedStrategyLibFacade is IPoolProportionsProvider {
     bool needToChange,
     PairBasedStrategyLib.FuseStatus status
   ) {
-    return PairBasedStrategyLib.needChangeFuseStatus(fuse_, price);
+    return PairBasedStrategyLib.needChangeFuseStatus(fuse_, price, price);
```

Comment:
Such fix can produce problems. Better to add new poolPrice param to the function declaration. The test will be broken and it will remain us to add new set of unit tests for needChangeFuseStatus (with price != poolPrice)